src/e2e-test/features/spanner/sink/GCSToSpanner.feature

```diff
@@ -1,14 +1,18 @@
 @Spanner_Sink @SPANNER_TEST
 Feature: Spanner Sink - Verification of GCS to Spanner data transfer
 
-  @GCS_CSV_TEST @SPANNER_SINK_TEST
+  @GCS_ALL_DATATYPES_FOR_SPANNER_SINK_TEST @SPANNER_SINK_TEST
```

Comment:
`Scenario: To verify data is getting transferred successfully from GCS to Spanner existing database new table`

Revert all changes done in this scenario as you have added new scenario for Spanner all datatype test

Also remove @GCS_ALL_DATATYPES_FOR_SPANNER_SINK_TEST  tag related changes from `TestSetupHooks`  and `pluginParameters.properties`